buildSrc/src/main/java/com/uber/okbuck/OkBuckGradlePlugin.java

```diff
@@ -81,7 +81,10 @@
   BuckManager buckManager;
 
   // Only apply to the root project
-  public void apply(@NotNull Project rootProject) {
+  @Initializer
+  @SuppressWarnings("NullAway")
```

Comment:
Shouldn't you only need one or the other?

Reply:
Will remove initializer

Reply:
Actually seems like I need both. Otherwise nullaway throws several errors instead of just the one
```
[NullAway] initializer method does not guarantee @NonNull fields manifestPath, packageName are initialized along all control-flow paths (remember to check for exceptions or early returns).
```

Which I am able to suppress with a more targeted and single method suppression